lib/wibox/widget/imagebox.lua

```diff
@@ -206,21 +219,25 @@ end
 
 --- Should the image be resized to fit into the available space?
 -- @property resize
--- @param allowed If false, the image will be clipped, else it will be resized
---   to fit into the available space.
+-- @tparam boolean resize
+
+--- Should the image be resized to fit into the available space?
+-- @tparam boolean allowed If `false`, the image will be clipped, else it will
+--   be resized to fit into the available space.
```

Comment:
I feel like "the image will be clipped" is slightly misleading. Yes, this is what happens when the image is larger than the available space. However, `resize = false` also has an effect when the image is smaller than the available space (the image is simply displayed as-is). No idea how to make this clearer...

(This also appears in the description of the constructor.)

Reply:
I think we can have an HTML table showing 3 use cases (image smaller than the widget, image size == widget size, and image bigger than the widget) for both values of the `resize` property.

But I'll need help on that. I'm not sure how this kind of tables are defined in other parts of the documentation.

(About the constructor, I think we need some changes on its implementation. When a property isn't explicitly given to it, the constructor will not set it at all! So `local ib = imagebox(beatiful.icon)` will result on `ib.resize == nil`. I didn't dig too much on that but I guess we can have errors depending on how it's used)

Reply:
I added an image example: `tests/examples/wibox/widget/imagebox/resize.lua`

![imagebox_resize](https://user-images.githubusercontent.com/6602958/69803531-b24ba280-11dc-11ea-8b22-20f54194e535.png)